recipes_source/xeon_run_cpu.rst

```diff
@@ -0,0 +1,302 @@
+Optimizing PyTorch Inference with Intel(R) Xeon(R) Scalable Processors
+======================================================================
+
+There are several configuration options that can impact the performance of PyTorch inference when executed on Intel(R) Xeon(R) Scalable Processors.
+To get peak performance, the ``torch.backends.xeon.run_cpu`` script is provided that optimizes the configuration of thread and memory management.
+For thread management, the script configures thread affinity and the preload of Intel(R) OMP library.
+For memory management, it configures NUMA binding and preloads optimized memory allocation libraries, such as TCMalloc and JeMalloc.
+In addition, the script provides tunable parameters for compute resource allocation in both single instance and multiple instance scenarios,
+helping the users try out an optimal coordination of resource utilization for the specific workloads.
+
+What You Will Learn
+-------------------
+
+* How to utilize tools like ``numactl``, ``taskset``, Intel(R) OpenMP Runtime Library and optimized memory allocators such as ``TCMalloc`` and ``JeMalloc`` for enhanced performance.
```

Comment:
```suggestion
* How to utilize tools like ``numactl``, ``taskset``, Intel(R) OpenMP Runtime Library and
   optimized memory allocators such as ``TCMalloc`` and ``JeMalloc`` for enhanced performance.
```

Reply:
Hi @svekars , it looks like we needn't separate these 2 sentences in 2 lines since they are bullets and have asterisks at the beginning.
With this change the output doc has 2 paragraphs for each bullet.
![image](https://github.com/pytorch/tutorials/assets/109502517/610a2bb3-df3c-44b2-957b-685f807a70d9)